src/recaptcha.js

```diff
@@ -11,8 +11,12 @@ export default class ReCAPTCHA extends React.Component {
   }
 
   getValue() {
-    if (this.props.grecaptcha && this._widgetId !== undefined) {
-      return this.props.grecaptcha.getResponse(this._widgetId);
+    let getResponse = this.props.grecaptcha && this.props.grecaptcha.getResponse;
+    if (this.props.grecaptcha.enterprise) {
+      getResponse = this.props.grecaptcha.enterprise.getResponse();
+    }
+    if (getResponse && this._widgetId !== undefined) {
+      return getResponse(this._widgetId);
     }
```

Comment:
IIUC, you're trying to figure out the method to call? I feel like this is more complex than it needs to be. The biggest condition is that the widget is undefined, so maybe rule that out first?

```suggestion
    if (this._widgetId === undefined || this.props.grecaptcha === undefined) {
      return null;
    }
    if (this.props.grecaptcha.enterprise) {
      return this.props.grecaptcha.enterprise.getResponse(this._widgetId);
    }
    return this.props.grecaptcha.getResponse(this._widgetId);
}
```

Either way, you may want to remove the parentheses on line 16, I think